gradle.properties

```diff
@@ -1,2 +1,5 @@
 kotlin.code.style=official
-kotlin.js.generate.executable.default=false
\ No newline at end of file
+kotlin.js.generate.executable.default=false
+
+klib_director.records_limit=4
```

Comment:
Just in case: usually spearator in `gradle.properties` file is `.`. It is not the rule, but something like habitable practice